sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/KalixMethod.scala

```diff
@@ -91,16 +91,36 @@ case class VirtualServiceMethod(component: Class[_], methodName: String, inputTy
   val streamOut: Boolean = false
 }
 
+case class CombinedSubscriptionServiceMethod(
+    combinedMethodName: String,
+    representative: SubscriptionServiceMethod,
+    inputClass2Method: Map[String, Method])
```

Comment:
I think this field should be called `methodsMap`. In any case, it's not a map from inputClass to Method, but from typeUrl to Method.

Wouldn't it be simpler if that map is defined as `Map[String, JavaMethod]`? I mean, if we wrap the method with its extractor earlier in the call chain, we don't need to look for it down on the road.